docs/pages/zero-trust-access/management/operations/breakglass-access.mdx

```diff
@@ -0,0 +1,433 @@
+---
+title: Configuring break-glass SSH Access for Disaster Recovery
+description: Guide to set up break-glass emergency SSH access for critical systems if Teleport becomes unavailable.
+---
+
+This guide will walk you through the steps required to configure emergency "break-glass" access for critical agents and servers via OpenSSH
+using Teleport-issued certificates, in the following scenarios:
+
+1. The Teleport Agent on a server has crashed, gone offline, or become unusable.
+2. The Teleport control plane is down and cannot be used to access systems, and out-of-band access is necessary to fix it.
+
+## How it works
+
+Teleport's `user` CA can issue short-lived, signed certificates that can be used to grant access to OpenSSH servers in emergency disaster recovery scenarios.
+
+By configuring the OpenSSH server alongside a Teleport Agent to trust Teleport's `user` CA, users with valid Teleport-issued certificates
+can authenticate to the server without requiring static SSH keys or passwords - even if Teleport itself is down.
+
+Below, we'll detail the steps to accomplish the following objectives:
+
+1. Configure the Teleport Agent's OpenSSH server to trust Teleport's `user` CA
+1. Create a `breakglass` system user on the remote host
+1. Restrict cert-based SSH access to this `breakglass` user only
+1. Create a `breakglass` Role in Teleport
+1. Create a Teleport Machine ID bot (`tbot`) to issue a `breakglass` SSH Key and Certificate using Teleport's CA on an ongoing basis
+1. Access the remote server using a Teleport issued cert, even if Teleport is down or unresponsive
+
+## Prerequisites
+
+- OpenSSH client version 7.4 or above on your local machine.
+- A Linux host with OpenSSH server (`sshd`) version 7.4 or above.
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+## Step 1/6. Configure sshd to trust the Teleport CA
+
+For break-glass access, the OpenSSH server must be configured to trust client certificates issued by the Teleport `user` Certificate Authority (CA).
+
+1. Export the public key of the Teleport `user` CA:
+
+**This step is completed on the Teleport Agent machine.**
+
+On the Teleport Agent machine, run the following command. Replace `teleport.example.com` with the address of your Teleport Proxy Service:
+
+```bash
+export KEY=$(curl 'https://<Var name="teleport.example.com" />/webapi/auth/export?type=user' | sed "s/cert-authority\ //")
+```
+
+2. Make the public key accessible to `sshd`:
+
+```bash
+echo "$KEY" | sudo tee /etc/ssh/teleport_user_ca.pub
+echo "TrustedUserCAKeys /etc/ssh/teleport_user_ca.pub" | sudo tee -a /etc/ssh/sshd_config
+```
```

Comment:
Appending a line to `sshd_config` is not guaranteed to work depending on whether any `Match` block was used.